package-lock.json

```diff
@@ -3582,6 +3573,15 @@
         }
       }
     },
+    "string_decoder": {
+      "version": "1.0.3",
+      "resolved": "https://registry.npmjs.org/string_decoder/-/string_decoder-1.0.3.tgz",
+      "integrity": "sha512-4AH6Z5fzNNBcH+6XDMfA/BTt87skxqJlO0lAh3Dker5zThcAxG6mKz+iGu308UKoPPQ8Dcqx/4JhujzltRa+hQ==",
+      "dev": true,
+      "requires": {
+        "safe-buffer": "5.1.1"
+      }
+    },
     "strip-ansi": {
       "version": "3.0.1",
       "resolved": "https://registry.npmjs.org/strip-ansi/-/strip-ansi-3.0.1.tgz",
```

Comment:
No need to edit this file.

Reply:
No I did not, it was automatic and I hate that lock file too. I can't control it.

Reply:
How could I avoid changing this lock file? I can not add it into .gitignore as this action will change your file, and everytime I run `npm install` it automatically edit it.